addons/stock/models/stock_picking.py

```diff
@@ -273,6 +273,9 @@ class Picking(models.Model):
         'Has Packages', compute='_compute_has_packages',
         help='Check the existence of destination packages on move lines')
 
+    entire_package_ids = fields.One2many('stock.quant.package', compute='_compute_entire_package_ids')
+    entire_package_detail_ids = fields.One2many('stock.quant.package', compute='_compute_entire_package_ids')
```

Comment:
add in a comment or in the docstring of the method why you need these two fields (displayed twice in the same formview)